src/app/tests/suites/certification/Test_TC_CC_3_1.yaml

```diff
@@ -206,61 +206,41 @@ tests:
               - name: "OptionsOverride"
                 value: 0
 
-    - label: "Wait 10s"
+    - label: "Wait 5s"
```

Comment:
my test plan copy says for 3c `After 10 seconds, TH reads CurrentHue attribute from DUT.`.

Is there some update in the test plan for this? Why did we switch to 5 seconds?

Reply:
I see, this assumes https://github.com/CHIP-Specifications/chip-test-plans/pull/4941 would be merged.